packages/hash/frontend/src/shared/layout/page-layout/plain-layout-with-header.tsx

```diff
@@ -0,0 +1,24 @@
+import Head from "next/head";
+import { ReactElement, ReactNode, VFC } from "react";
+import { isProduction } from "../../../lib/config";
+import { PageHeader } from "../../layout";
+
+const PlainLayoutWithHeader: VFC<{
+  children?: ReactNode;
+}> = ({ children }) => {
+  return (
+    <>
+      <Head>
+        <title>HASH Workspace</title>
+        <link rel="icon" type="image/png" href="/favicon.png" />
+        {!isProduction ? <meta name="robots" content="noindex" /> : null}
+      </Head>
+      <PageHeader />
+      {children}
+    </>
```

Comment:
```suggestion
    <PlainLayout>
      <PageHeader />
      {children}
    </PlainLayout>
```

Reply:
Addressed in => [8c19e64](https://github.com/hashintel/hash/pull/468/commits/8c19e64f44905c778cdeca75b7c8d919b5e779b3)